test/kafka-ingest-open-loop/mzcompose.py

```diff
@@ -109,6 +114,12 @@ def workflow_kafka_ingest_open_loop(
             "--disable-persistent-system-tables-test",
         ]
 
+    if args.s3_storage is not None:
+        options.extend([
```

Comment:
I'd love for this to error if the flag is set but empty so someone can't accidentally pollute the top-level namespace of the bucket but I didn't see any examples of mzworkflows that do that sort of validation and nothing in the argparse docs jumped out at me

Reply:
Yeah the `argparse` way is just to do that validation yourself:

```python
if args.s3_storage == "":
    print("--s3-storage value must be non-empty", file=sys.stderr)
    sys.exit(1)
elif args.s3_storage:
    options.extend(...)
```

Reply:
done. ty!